diff -up cantata-2.4.2/replaygain/ffmpeginput.cpp.omv~ cantata-2.4.2/replaygain/ffmpeginput.cpp
--- cantata-2.4.2/replaygain/ffmpeginput.cpp.omv~	2022-01-16 14:19:21.189453979 +0100
+++ cantata-2.4.2/replaygain/ffmpeginput.cpp	2022-01-16 14:21:05.921018076 +0100
@@ -106,7 +106,7 @@ struct FfmpegInput::Handle {
     }
     AVFormatContext *formatContext;
     AVCodecContext *codecContext;
-    AVCodec *codec;
+    const AVCodec *codec;
     #if LIBAVCODEC_VERSION_INT >= AV_VERSION_INT(53, 35, 0)
     AVFrame *frame;
     int gotFrame;
@@ -207,7 +207,7 @@ FfmpegInput::FfmpegInput(const QString &
 
         if (ok) {
             QString floatCodec=QLatin1String(handle->codec->name)+QLatin1String("float");
-            AVCodec *possibleFloatCodec = avcodec_find_decoder_by_name(floatCodec.toLatin1().constData());
+            const AVCodec *possibleFloatCodec = avcodec_find_decoder_by_name(floatCodec.toLatin1().constData());
             if (possibleFloatCodec) {
                 handle->codec = possibleFloatCodec;
                 #if LIBAVCODEC_VERSION_INT >= AV_VERSION_INT(57, 89, 100) // Not 100% of version here!
