--- cantata-2.2.0/replaygain/ffmpeginput.cpp~	2017-04-07 19:04:12.000000000 +0100
+++ cantata-2.2.0/replaygain/ffmpeginput.cpp	2017-12-28 17:52:50.938999892 +0000
@@ -40,7 +40,7 @@
 #define AVCODEC_MAX_AUDIO_FRAME_SIZE 192000 // 1 second of 48khz 32bit audio
 #endif
 
-#define BUFFER_SIZE ((((AVCODEC_MAX_AUDIO_FRAME_SIZE * 3) / 2) * sizeof(int16_t)) + FF_INPUT_BUFFER_PADDING_SIZE)
+#define BUFFER_SIZE ((((AVCODEC_MAX_AUDIO_FRAME_SIZE * 3) / 2) * sizeof(int16_t)) + AV_INPUT_BUFFER_PADDING_SIZE)
 
 #if LIBAVCODEC_VERSION_INT >= AV_VERSION_INT(57, 24, 0)
 #define AV_FREE(PKT) av_packet_unref(PKT)
